packages/terra-switch/tests/wdio/switch-spec.js

```diff
@@ -0,0 +1,45 @@
+Terra.describeViewports('Switch', ['tiny', 'medium', 'large'], () => {
```

Comment:
I think we only need to test in tiny. It doesn't appear the switch stretches across the screen or is responsive to the view port width.
```suggestion
Terra.describeViewports('Switch', ['tiny'], () => {
```

Reply:
It does look like the long label test wold want to test in multiple viewports but the rest wouldn't need to.

Reply:
changed spec file to test for tiny viewport and other viewports for labelwrap test : [here](https://github.com/cerner/terra-core/pull/2846/commits/8ba4fb42d6a30245f9e12acc308252d47b22d7d2#diff-281655daa14a9bf8a5739b0a51185b24)